packages/commerce-sdk-react/src/auth/storage.ts

```diff
@@ -6,10 +6,26 @@
  */
 import Cookies from 'js-cookie'
 
-export interface BaseStorage {
-    set(key: string, value: string, options?: unknown): void
-    get(key: string): string
-    delete(key: string): void
+export type StorageTypes = 'cookie' | 'local'
```

Comment:
🤔 We could add `static type: StorageType` to each implementation. That could potentially enable some fancy type magic. Not sure we have a use case for fancy type magic here, but maybe someone downstream will...